src/components/core/PivotTable.tsx

```diff
@@ -638,31 +680,40 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         if (!this.isGrowToFitEnabled()) {
             return;
         }
+
+        invariant(columnApi !== undefined, "calling grow to fit without column api cannot work");
```

Comment:
growToFit can be called from componentDidUpdate when widthDefs changed, and because this can happen even before ag-grid sends its first event gridReady, where we gather columnApi I wanted to add some check. Its up to us if this will be invariant of silent return